collections/_article/parallax-progress-report.md

```diff
@@ -0,0 +1,112 @@
+
+You may notice that `Parallax2D`'s `follow_viewport` is enabled by default. This decision was made because `ParallaxBackground`'s `follow_viewport_enabled` is *not* enabled even though it *is* following the viewport. It can be confusing for a `CanvasLayer` to not behave like a `CanvasLayer`, so instead `Parallax2D` behaves more like one would expect.
+
+#### `autoscroll`
+
+`autoscroll` scrolls the layer automatically in addition to the parallax settings. Honestly, that's it! This could be done with a custom script, but we decided to add it since it was highly requested, easy to implement, and has no performance hit if you aren't using it.
+
+#### `repeat_times`
+
+This shouldn't come up often, but the new `repeat_times` property is being introduced to help with the problem of zooming out while trying to achieve an infinite repeat effect. When you zoom out and the repeat size becomes smaller than the screen, the effect is broken. In the video below, both scenes depict a collection of scrolling `Parallax2D` nodes while zoomed out. The top scene has their `repeat_times` set to 1, and the bottom scene is set to 3 (an extra repeat to the left and right).
+
+<video autoplay loop muted playsinline title="An demonstration of the parallax effect zoomed out">
+  <source src="/storage/blog/parallax2d/zoom_out.mp4" type="video/mp4">
+</video>
+
+Increasing the `repeat_times` will allow for the number of repeats to grow and spread outward, so that you can zoom out and not reveal the magic going on behind the scenes. Eventually, it'd be nice to have a flag to adjust this automatically, but this is a good start! Bear in mind that this repeats *all* children of the `Parallax2D` node. If you want only one of the children to repeat, it might be preferable to use the `texture_repeat` property and `Sprite2D`'s region to prepare your background.
+
+#### What else?
+
+`Parallax2D` is now compatible with `BackBufferCopy` and `CanvasGroup`, and you're also free to rotate the camera without breaking the parallax effect! Upon a few initial tests, we noticed some performance increases as well.
+
+### Caveats and future
+
+With any effect that is based on perception, there are some tricky areas (since the entire effect is a trick!). A few of the same drawbacks to `ParallaxBackground` still apply to `Parallax2D`. For example, multiple cameras aren't directly supported (booo!). This makes sense because `Parallax2D` follows the same concept as `ParallaxBackground`: the textures' positions are modified directly and move at different speeds relative to the viewport. If you have multiple viewports, the textures can't move relative to all of them and can't be in two places at once! Just like in `ParallaxBackground`, as a workaround, you can clone the `Parallax2D`s and place each in a separate viewport.
```

Comment:
> (booo!)

Again, the sentence leaves a better impression on the reader without the interjection taking away from it, so I would cull it.